locales/fr/LC_MESSAGES/base.po

```diff
@@ -22,87 +22,140 @@ msgstr ""
 #: bot.py:58
```

Comment:
looks like this part was missing in the originall file : 
#: bot.py:56
msgid "ECO_TITLE"
msgstr "Eco"

Could you update the PR with this @siouxx2021  ?